.eslintrc

```diff
@@ -30,7 +30,7 @@
         "generator-star-spacing": ["error", "before"],
         "indent": ["error", 4],
         "linebreak-style": ["error", "unix"],
-        "max-len": ["error", 120, 2],
+        "max-len": ["error", 150, 2],
```

Comment:
We should bump .prettierrc printWidth as well to keep it consistent

Reply:
Added to .prettierrc